astrodendro/structure.py

```diff
@@ -78,7 +78,10 @@ def __init__(self, indices, values, children=[], idx=None, dendrogram=None):
             self._values = [x for x in values]
             self._vmin, self._vmax = min(values), max(values)
 
-        self._smallest_index = min(self._indices)
+        try:
+            self._smallest_index = min(self._indices)
+        except ValueError:
```

Comment:
What causes the ValueError in min? Is there a check somewhere if `_indices` is empty?